src/components/ComplexLookupField/ModalComplexLookup.tsx

```diff
@@ -1,26 +1,53 @@
-import { FC, memo } from 'react';
+import { FC, memo, useCallback } from 'react';
+import { SEARCH_API_ENDPOINT } from '@common/constants/api.constants';
 import { Modal } from '@components/Modal';
+import { Search } from '@components/Search';
+import { SearchControlPane } from '@components/SearchControlPane';
+import { filters } from './data/filters';
 import './ModalComplexLookup.scss';
 
-interface Props {
+interface ModalComplexLookupProps {
   isOpen: boolean;
   onClose: VoidFunction;
   title?: string;
+  searchPaneTitle?: string;
 }
 
-export const ModalComplexLookup: FC<Props> = memo(({ isOpen, onClose, title = '' }) => {
-  return (
-    <Modal
-      isOpen={isOpen}
-      title={title}
-      onClose={onClose}
-      titleClassName="modal-complex-lookup-title"
-      showModalControls={false}
-    >
-      <div className="complex-lookup-search-contents" data-testid="complex-lookup-search-contents">
-        {/* TODO: add the required content */}
-        <p>Modal content</p>
-      </div>
-    </Modal>
-  );
-});
+export const ModalComplexLookup: FC<ModalComplexLookupProps> = memo(
+  ({ isOpen, onClose, title = '', searchPaneTitle = '' }) => {
+    const renderSearchControlPane = useCallback(() => <SearchControlPane label={searchPaneTitle} />, []);
+    // TODO: create a component
+    const renderResultsList = useCallback(() => <div />, []);
+
+    return (
+      <Modal
+        isOpen={isOpen}
+        title={title}
+        onClose={onClose}
+        titleClassName="modal-complex-lookup-title"
+        showModalControls={false}
+        className="modal-complex-lookup"
+        classNameHeader="modal-complex-lookup-header"
+      >
+        <div className="complex-lookup-search-contents" data-testid="complex-lookup-search-contents">
+          <Search
+            // TODO: Extend the profile with the endpoint and use it here
+            endpointUrl={`${SEARCH_API_ENDPOINT}/authorities`}
```

Comment:
minor - it may be better to define this as a constant in [api.constants.ts](https://github.com/FOLIO-EIS/ui-linked-data/pull/289/files#diff-da69f6b229bad82e6d2bd0866ee6e74b1dba5655e895598a778654aff3b1bf1a)

Reply:
This is a temporary solution and I added TODO comment here.
As different Complex lookups may have different API endpoints, we should determine whether it is better to store them in the profile or in a map data structure on the FE side. This is the scope of the further tickets.